docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-principal-group-name.adoc

```diff
@@ -1,9 +1,87 @@
-[[configure-dn-parts-used-for-groups]]
-= Configure Using Distinguished Name Parts for Groups
+[[certificate-realm-principal-name-group-names]]
+= Certificate Realm: Principal Name, Group Names
+
+[[configure-the-principal-name-on-certificate-realms]]
+== Configure the Principal Name on Certificate Realms
+
+Payara Server Community includes a feature that supports using exclusively the use of the Common-Name (`CN`) from a client
```

Comment:
```suggestion
Payara Server includes a feature that supports using exclusively the use of the Common-Name (`CN`) from a client
```

Reply:
It will be easier to keep it platform agnostic, as the same information is in Enterprise - porting it will be easier in the future.